tests/resources/promise-rejection-events.js

```diff
@@ -464,47 +464,87 @@ async_test(function(t) {
     var unreached = t.unreached_func('promise should not be fulfilled');
     p.then(unreached, function() {});
   });
-}, 'delayed handling: a task delay before attaching a handler does not prevent unhandledrejection');
+}, 'delayed handling: a task delay before attaching a handler prevents unhandledrejection');
```

Comment:
why? The microtask checkpoint already scheduled a task to report the event. When the handler is added, the rejected promise is no longer in the list of about to be notified promises, so we'll just queue a task to notify about the addition of the handler as well.